src/test/java/com/sap/sgs/phosphor/fosstars/model/score/oss/VulnerabilityLifetimeScoreTest.java

```diff
@@ -1,43 +1,72 @@
 package com.sap.sgs.phosphor.fosstars.model.score.oss;
 
 import static com.sap.sgs.phosphor.fosstars.TestUtils.assertScore;
-import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.FIRST_COMMIT_DATE;
 import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.PROJECT_START_DATE;
 import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.VULNERABILITIES;
 import static com.sap.sgs.phosphor.fosstars.model.other.Utils.setOf;
+import static org.junit.Assert.assertEquals;
 
 import com.sap.sgs.phosphor.fosstars.model.Score;
 import com.sap.sgs.phosphor.fosstars.model.Value;
 import com.sap.sgs.phosphor.fosstars.model.other.Utils;
+import com.sap.sgs.phosphor.fosstars.model.value.CVSS;
 import com.sap.sgs.phosphor.fosstars.model.value.Vulnerabilities;
+import com.sap.sgs.phosphor.fosstars.model.value.Vulnerability;
+import com.sap.sgs.phosphor.fosstars.model.value.Vulnerability.Resolution;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Duration;
+import java.util.Collections;
 import java.util.Date;
 import java.util.Set;
 import org.junit.Test;
 
 public class VulnerabilityLifetimeScoreTest {
 
+  private static final double PRECISION = 0.01;
+
+  private static final VulnerabilityLifetimeScore VULNERABILITY_LIFETIME_SCORE
+      = new VulnerabilityLifetimeScore();
+
+  private static final SimpleDateFormat DATE_FORMAT
+      = new SimpleDateFormat("dd/MM/yyyy");
```

Comment:
Would it be better to have a constant LOCALE.US?

Reply:
I am not sure if it's a problem here but let's update it to use a specific locale to be on the safe side.